common/src/test/java/org/apache/nemo/common/DAGTest.java

```diff
@@ -105,7 +105,7 @@ public void testNormalDAG() {
 
     final List<IntegerVertex> topologicalOrder = dag.getTopologicalSort();
     assertEquals(topologicalOrder.get(0).getValue(), 4);
```

Comment:
I believe that this was also the line that you were meant to change. Please check and confirm! Thanks :)

Reply:
https://sonarcloud.io/project/issues?id=org.apache.nemo%3Anemo-project&issues=AWkltJL0gtFwgfOLu2S2&open=AWkltJL0gtFwgfOLu2S2